examples/simple/lint/L-EXC3-bad.c

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2015 Runtime Verification, Inc. (RV-Match team). All Rights Reserved.
+
+#include <stdlib.h>
+#include <stdio.h>
+
+void func(int number)
+{
+    printf("func called with number: %i\n", number);
+}
+
+int main()
+{
+    void (*func_ptr)(int) = &func;
+    if(func_ptr != ((void *) NULL))
```

Comment:
also I believe the idea behind this error is it occurs with functions that take no arguments when the user has forgotten to add the () that causes the function to actually be invoked. So I think the good example here should be that we are actually calling func().